src/views/HomeView.vue

```diff
@@ -39,29 +39,41 @@
               variant="outlined"
               color="primary"
               class="mt-4"
-              :small="$vuetify.display.mdAndDown ? true : false"
               to="/rules"
               data-cy="rules-link"
+              v-bind="{
+                ...($vuetify.display.mdAndDown ? {
+                  size: 'small',
+                } : {})
+              }"
             >
               Rules
             </v-btn>
             <v-btn
               variant="outlined"
               color="secondary"
               class="mt-4"
-              :small="$vuetify.display.mdAndDown ? true : false"
               href="https://human-ai-interaction.github.io/cuttle-bot/"
               target="_blank"
               data-cy="ai-link"
+              v-bind="{
+                ...($vuetify.display.mdAndDown ? {
+                  size: 'small',
+                } : {})
+              }"
             >
               Play with AI
             </v-btn>
             <v-btn
               variant="outlined"
               class="mt-4"
-              :small="$vuetify.display.mdAndDown"
               href="https://discord.gg/9vrAZ8xGyh"
               target="_blank"
+              v-bind="{
+                ...($vuetify.display.mdAndDown ? {
+                  size: 'small',
+                } : {})
+              }"
```

Comment:
Seeing this 3 times in this file makes me think we want a computed prop for the v-bind object